### ISSUE TITLE
IPv6 Binding problem

### ISSUE THREAD

**Issue by albertotombax** [kgretzky/evilginx2#564]
Hi, 

I have problem with binding server only for IPv6 : 
netstat : 

tcp6       0      0 [::]:http               [::]:*                  LISTEN      5901/evilginx        off (0.00/0/0)
tcp6       0      0 [::]:https              [::]:*                  LISTEN      5901/evilginx        off (0.00/0/0)

I tried all methods in config and disable IPv6 in system. Where can I disable ipv6 or set binding IP?

**Comment by albertotombax** [comment]
I think this is problem : https://github.com/golang/go/issues/5197 

replace `net.Listen("tcp",` to `net.Listen("tcp4",` in source partially solve problem.

**Comment by pawp81** [comment]
How did you completely fix it? I edited http_proxy.go and replaced net.Listen("tcp" with net.Listen("tcp4". Then I recompiled go source code with "make". After starting evilginx port 443 uses Ipv4, but port 80 uses Ipv6
tcp 0 0 0.0.0.0:22 0.0.0.0:* LISTEN 935/sshd
tcp 0 0 0.0.0.0:443 0.0.0.0:* LISTEN 4610/evilginx
tcp6 0 0 :::80 :::* LISTEN 4610/evilginx
tcp6 0 0 :::22 :::* LISTEN 935/sshd
udp6 0 0 :::53 :::* 4610/evilginx

Which makes acme to fail:
 acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://my.Url.com/.well-known/acme-challenge/3qviInEMYtOu_ZsRlhGu5zoKxlUVndEgWjrUIv_w-Ts: Timeout during connect (likely firewall problem)
